db/schema.rb

```diff
@@ -53,9 +53,8 @@
     t.integer  "form_id"
     t.integer  "question_id"
     t.integer  "response_set_id"
-    t.datetime "created_at",                  null: false
-    t.datetime "updated_at",                  null: false
-    t.integer  "sort",            default: 0
```

Comment:
Honestly, because I had to rebase it on something that changed schema and it changed spacing? 

As for why it's dropping sort - there's no schema to do it, I'll take a quick look at it.